DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -219,6 +224,10 @@ export class SearchSortFilterAndPaginate {
   }
 
   private updatePageNumber(pageNumber: number, searchableData: ISearchableData): void {
+    if (this.paginationEnabled === false) {

```

Comment:
Could we include this condition in the method name?
`updatePageNumberIfPaginated`?